src/components/Toolbelt/Toolbelt.test.js

```diff
@@ -12,36 +14,58 @@ jest.mock('../../config', () => ({
   },
 }))
 
+jest.mock('../../utils', () => ({
+  ...jest.requireActual('../../utils'),
+  memoize: jest.fn(callback => {
+    return (...args) => {
+      return callback(...args)
+    }
+  }),
```

Comment:
This is a great way to test memoized functions. We should use this more often! :clap: